src/frontend/js/components/Dashboard/DashboardBreadcrumbs.tsx

```diff
@@ -0,0 +1,123 @@
+  {} as any,
+);
+
+/**
+ * Provider that needs to wrap any component using the DashboardBreadcrumbs component.
+ * It provides a context used to make formatjs value available bottom up.
+ */
+export const DashboardBreadcrumbsProvider: FC = ({ children }) => {
+  const [placeholders, setPlaceholders] = useState<DashboardBreadcrumbsPlaceholders>({});
+
+  const pushBreadcrumbsPlaceholders = (data: DashboardBreadcrumbsPlaceholders) => {
+    setPlaceholders({ ...placeholders, ...data });
+  };
+
+  // JSON.stringify is needed because `placeholders` is an object this is the same as doing a
+  // shallow equality test. Otherwise, it will create an infinite re-rendering loop as the object
+  // reference changes on each use of pushBreadcrumbsPlaceholders.
+  const context = useMemo(
+    () => ({
+      pushBreadcrumbsPlaceholders,
+      meta: { placeholders },
+    }),
+    [JSON.stringify(placeholders)],
+  );
+
+  return (
+    <DashboardBreadcrumbsContext.Provider value={context}>
+      {children}
+    </DashboardBreadcrumbsContext.Provider>
+  );
+};
+
+/**
+ * Hook to be used in every component that needs to provide a formatjs value to breadcrumbs's part.
+ */
+export const useBreadcrumbsPlaceholders = (data: DashboardBreadcrumbsPlaceholders) => {
+  const context = useContext(DashboardBreadcrumbsContext);
+  useEffect(() => {
+    context.pushBreadcrumbsPlaceholders(data);
+  }, [data]);
+};
+
+/**
+ * Returns formatted breadcrumbs parts.
+ */
+const useBreadcrumbsParts = () => {
+  const context = useContext(DashboardBreadcrumbsContext);
+  const matches = useMatches();
+  const intl = useIntl();
+  return matches
+    .filter((match) => (match.handle as any)?.crumbLabel)
+    .filter((match) => {
+      const { crumbLabel } = match.handle as { crumbLabel: MessageDescriptor };
+      return IntlHelper.doValuesExists(crumbLabel, context.meta.placeholders);
+    })
+    .map((match) => {
+      const { crumbLabel } = match.handle as { crumbLabel: MessageDescriptor };
+      return {
+        pathname: match.pathname,
+        name: intl.formatMessage(crumbLabel, context.meta.placeholders),
+      };
+    });
+};
+
+export const DashboardBreadcrumbs = () => {
+  const navigate = useNavigate();
+  const breadcrumbs = useBreadcrumbsParts();
+
+  const back = () => {
+    if (breadcrumbs.length <= 1) {
+      location.replace('/');
+      return;
+    }
+    navigate(breadcrumbs[breadcrumbs.length - 2].pathname);
+  };
+
+  return (
+    <ul className="dashboard-breadcrumbs" data-testid="dashboard-breadcrumbs">
+      <li>
+        <Button onClick={back} size="tiny">
+          <Icon name="icon-chevron-left" className="button__icon" />
+          <FormattedMessage {...messages.back} />
+        </Button>
+      </li>
+
+      {breadcrumbs.map((breadcrumb) => (
+        <li key={breadcrumb.pathname}>
+          <Button onClick={() => navigate(breadcrumb.pathname)} size="tiny">
```

Comment:
I think there is an a11y issue to use a button here. This element should appear as a link.
```
          <Link to={breadcrumb.pathname} size="tiny">
```